src/server/index.d.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+export interface KbnServer {
+    config: () => configObject;
+  }
+  
+export interface configObject {
+    get: (path: string) => any;
+}
```

Comment:
hm, the aligning in this file didn't get auto corrected.  Maybe because of the `.d.ts` extension. 

can you update so it's only two space indentations?  and the `}` on line 8 should have no spaces before it.